release/release_tests.yaml

```diff
@@ -5713,6 +5713,30 @@
       cluster:
         cluster_compute: single_node_benchmark_compute_gce.yaml
 
+- name: read_images_benchmark_multi_node
+  group: data-tests
+  working_dir: nightly_tests/dataset
+
+  frequency: nightly
+  team: data
+
+  cluster:
+    byod:
+      type: gpu
+    cluster_compute: multi_node_benchmark_compute.yaml
+
+  run:
+    timeout: 28800
```

Comment:
Is this one taking 8 hours to finish? That looks too long, which would make oncall debugging hard when it has some failure later. Should we make this one just run weekly, instead of daily, and add more nodes? Currently it uses 4 m5.4xlarge according to yaml file.

Reply:
It'll definitely take more than a couple hours, but I'm not sure how long it'll exactly take in total. I'll update the timeout accordingly before we merge this in

I updated the compute to use 20 m5.4xlarge and the frequency to 3x a week. Ultimately, the issue isn't that we don't have enough resources, but rather that the code is poorly optimized.